src/main/java/uk/gov/hmcts/reform/sscs/factory/CcdNotificationWrapper.java

```diff
@@ -113,7 +113,8 @@ public SscsCaseData getOldSscsCaseData() {
     private boolean hasAppointee() {
         return (responseWrapper.getNewSscsCaseData().getAppeal() != null
             && responseWrapper.getNewSscsCaseData().getAppeal().getAppellant() != null
-            && responseWrapper.getNewSscsCaseData().getAppeal().getAppellant().getAppointee() != null);
+            && responseWrapper.getNewSscsCaseData().getAppeal().getAppellant().getAppointee() != null
```

Comment:
This condition was failing because its returning the appointee object with values null
i.e Appointee(name=Name(title=null, firstName=null, lastName=null), address=Address(line1=null, line2=null, town=null, county=null, postcode=null), contact=Contact(email=null, phone=null, mobile=null), identity=Identity(dob=null, nino=null))